### ISSUE TITLE
apimon slows down Windows 8.1Update 1 x64

### ISSUE THREAD

**Issue by skvl** [tklengyel/drakvuf#1011]
For some reason scanning with dll-hooks.txt like this:
```
kernel32.dll,GetModuleFileNameA,log,hModule:hmodule,lpFilename:lpstr,nSize:dword
kernel32.dll,GetModuleFileNameW,log,hModule:hmodule,lpFilename:lpstr,nSize:dword
```
, slows down the Windows several times.

This could be clearly seen with injection of _EXCEL.EXE_

**Comment by icedevml** [comment]
@skvl So you've tested that without just these two hooks it works few times faster?

**Comment by skvl** [comment]
@icedevml yes. You are right. This is not a big deal to me at the time. I've just noticed that interesting fact.

**Comment by icedevml** [comment]
:thinking: This is worth inspecting. Right now I can't even guess why it could be like that. Thanks for reporting.

**Comment by kaboreka** [comment]
im facing with this issue
@icedevml did you have any thoughts on this issue?
how speed up guest host while monitoring
